### PR TITLE
[Wallet] Upgrade react-native-webview to 11.0.2

### PR DESCRIPTION
**Pull request by etuleu**
### Description

Upgraded the react-native-webview library to mitigate [CVE-2020-6506](https://github.com/advisories/GHSA-36j3-xxf7-4pqg). [Release notes.](https://github.com/react-native-webview/react-native-webview/releases/tag/v11.0.0 )

### Other changes

Cleared the audit file.

### Tested

Manually and e2e

### Related issues

- Fixes #[issue number here]

### Backwards compatibility

Yes.

### CODE REVIEW

File: packages/mobile/ios/Podfile.lock [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PODS:
-  - Analytics (4.1.1)
+  - Analytics (4.1.2)
```
Comment (jeanregisser) [inline packages/mobile/ios/Podfile.lock]: We shouldn't upgrade unrelated things in this PR.

**Review by jeanregisser — COMMENTED** [review]
Looks good! See comment below.

File: packages/mobile/ios/Podfile.lock (hunk 1 above)
Comment (etuleu) [reply packages/mobile/ios/Podfile.lock]: Thanks! I did a pod update but not sure how this happened also. Reverting

Comment (etuleu) [reply packages/mobile/ios/Podfile.lock]: Fixed

**Review by jeanregisser — APPROVED** [review]
🙏

**Pull request merged**